include/main.h

```diff
@@ -14,4 +14,6 @@ constexpr array<string_view, 2> config_dir = {".config", "lmk"};
 #endif
 constexpr string_view histfile = ".lmkhistory";
 
+constexpr string_view dbDir = "./database/";
```

Comment:
question whether we should use a single database/ folder in the user's home dir. 
This would mean we'd have access to the same tables from anywhere in our filesystem.

Reply:
makes sense